mcproxy/patches/0003-add-sourcefilter.patch

```diff
@@ -4,7 +4,7 @@
  #include <numeric>
  #include <unistd.h>
  
-+#ifndef __GLIBC__
++#if !defined(__GLIBC__) || defined(__UCLIBC__)
```

Comment:
Is this really supposed to mean "not glibc or uclibc"? Or did you rather mean `#if  !defined(__GLIBC__) && !defined(__UCLIBC__)` ?

Reply:
I haven't tested this, but reading the commit message it looks correct.  
uClibc-ng defines `__GLIBC__` but needs to include `sourcefilter.cpp`.

Reply:
It's correct. It's needed for uclibc-ng but not glibc.